build-tests/heft-jest-reporters-test/config/typescript.json

```diff
@@ -28,19 +28,25 @@
     //    */
     //    "outFolderName": "lib-amd"
     // }
-    {
-      "moduleKind": "commonjs",
-      "outFolderName": "lib-commonjs"
-    }
   ],
 
+  /**
+   * If true, will emit CommonJS module output to \"lib\" with the .cjs extension alongside (or instead of, if TSConfig specifies CommonJS) the default compilation output.
+   */
+  "emitCjsExtensionForCommonJS": true,
+
+  /**
+   * If true, will emit ESNext module output to \"lib\" with the .mjs extension alongside (or instead of, if TSConfig specifies ESNext) the default compilation output.
```

Comment:
```suggestion
   * If true, emit ESNext module output to the folder specified in the tsconfig "outDir" compiler option with the .mjs extension alongside (or instead of, if TSConfig specifies ESNext) the default compilation output.
```